app/views/sessions/_filters.html.erb

```diff
@@ -60,7 +60,7 @@
                 <%= form.submit(
                   t("views.status_filters.submit"),
                   name: "",
-                  class: "text-lg text-red italic underline font-black"
+                  class: "text-lg text-red italic underline font-black hover:cursor-pointer"
```

Comment:
Improvements:
1. The pointer cursor should also be displayed when hovering over the whole button, not just the SVG.
2. The menu should be displayed when clicking anywhere on the button, not just the SVG
3. In the line `27` Use the format for conditional classes using a `content_tag`, instead of string interpolation i.e
```
class: [
          "flex min-w-fit items-center justify-center px-4 py-2 font-medium rounded-full",
          "bg-red text-white": params[:starts_at] == session_date.to_s,
          "bg-white text-red": params[:starts_at] != session_date.to_s
        ]
```

Reply:
@Luigi it seems all these changes have been applied.